opera/rules.go

```diff
@@ -181,11 +181,11 @@ type EconomyRules struct {
 
 	// MinGasPrice defines a lower boundary for the gas price
 	// on the network. However, its interpretation is different
-	// in the context of the Fantom and Sonic networks.
+	// in the context of the Sonic and Sonic networks.
```

Comment:
```suggestion
	// in the context of the Sonic network.
```
Or keep the original (maybe in reverse order) as it is also the case for the fantom network.